Logger.go

```diff
@@ -149,6 +157,14 @@ func getFrameInfo() frameInfo {
 	}
 }
 
+func getRequestId(ctx context.Context) string {
+	requestId, ok := ctx.Value(key).(string)
+	if !ok {
+		return "id not available"
```

Comment:
Magic string; het zou mooier zijn als de methode null of een sentinel value returned. En bij createEntry ook alleen loggen als er een requestId gevonden is.
```suggestion
		return "", false
```